package.json

```diff
@@ -44,6 +44,7 @@
     "lucene": "^2.1.0",
     "moment": "^2.29.4",
     "mutationobserver-shim": "^0.3.7",
+    "path": "^0.12.7",
```

Comment:
I wouldn't use [path](https://github.com/jinder/path) since it's poorly maintained and not tested. You might want to use the Browserify variant instead: https://github.com/browserify/path-browserify

Reply:
Yeah but sadly "path-browserify only implements the POSIX functions, not the win32 ones."